src/course-unit/course-sequence/sequence-navigation/SequenceNavigationTabs.jsx

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+import { Plus as PlusIcon } from '@edx/paragon/icons';
+import { Link } from 'react-router-dom';
+import { useIntl } from '@edx/frontend-platform/i18n';
```

Comment:
[optional] move this import above `@edx/paragon` imports to have structure like
```suggestion
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';
import { Button } from '@edx/paragon';
import { Plus as PlusIcon } from '@edx/paragon/icons';
import { useIntl } from '@edx/frontend-platform/i18n';
```